functions/AutoRemediateCT-001.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+
+/**
+* Lambda function to automatically remediate CloudTrail not Enabled
+*/
+module.exports.handler = (event, context, callback) => {
+
+	console.log('CloudTrail not Global - Received event:', JSON.stringify(event, null, 2));
+
+	if (!event || !event.region || event.ruleId !== "CT-001") {
+		return handleError("Invalid event");
+	}
+
+	var cloudtrail = new AWS.CloudTrail({apiVersion: '2013-11-01'});
+	cloudtrail.updateTrail({
+
+		Name: 'GlobalTrail',
+		S3BucketName: 'cc-remediate-cloudtrail',
+		// CloudWatchLogsLogGroupArn: 'STRING_VALUE',
+		// CloudWatchLogsRoleArn: 'STRING_VALUE',
+		IncludeGlobalServiceEvents: true,
+		IsMultiRegionTrail: true ,
+		// KmsKeyId: 'STRING_VALUE',
+		S3KeyPrefix: 'cloudtrail-global',
```

Comment:
In my opinion, it'd be better to leave S3KeyPrefix empty - which is usually the case; (or make it configurable)